h2o-algos/src/main/java/hex/DataInfo.java

```diff
@@ -808,6 +809,57 @@ public final int nextNumericIdx(int currentColIdx) {
     return res;
   }
 
+  public final int[] coefOriginalColumnIndices() {
```

Comment:
This look like a copy&paste&modify of something, is it?

Reply:
Yes, it is modified `coefNames()` method. However, I checked the meaning of every lines to be sure, it makes sense. But there probably could be one method for precalculation of both _coefNames and _coefOriginalColumnIndices...

Reply:
> But there probably could be one method for precalculation of both _coefNames and _coefOriginalColumnIndices...

I would rather keep it on the side, maybe if we moved it out of DataInfo it could be significantly simplified? A lot of logic handled InteractionWrappedVecs which don't make sense in case of tree models.